test/loader.spec.js

```diff
@@ -369,7 +369,7 @@ describe('loader', () => {
 					function(err, code, map) {
 						expect(err).not.to.exist;
 
-						expect(code).to.contain(require.resolve('../lib/hot-api.js').replace(/[/\\]/g, '/'));
+						expect(code).to.contain(`lib/hot-api.js`);
```

Comment:
use single quotes instead of backticks (if we apply svelte's linting to this repo at some point in the future that will be enforced. doing it now will make it easier to possibly setup)